### ISSUE TITLE
cmake / build files syntax

### ISSUE THREAD

**Issue by RobertLemmens** [slynch8/10x#660]
First of all i love 10x and i appreciate your work.

Are there any plans to add syntax highlights for popular build tools like cmake? Or maybe a system that  allows for users to implement these themself? I do use cmake and the files are a bit hard to wrap my head around without syntax highlights!

Thanks!

**Comment by slynch8** [comment]
Yes, I've been thinking of adding a general syntax highlighting system which will be customizable. Hopefully I'll get to it soon.

**Comment by castano** [comment]
+1
